diff --git a/node_modules/react-native-floating-label-input/src/index.tsx b/node_modules/react-native-floating-label-input/src/index.tsx
index 7b2733b..44215e5 100644
--- a/node_modules/react-native-floating-label-input/src/index.tsx
+++ b/node_modules/react-native-floating-label-input/src/index.tsx
@@ -19,15 +19,23 @@ import {
   TouchableWithoutFeedback,
   LayoutChangeEvent,
 } from 'react-native';
-import Animated, { EasingNode, timing, interpolateColors } from 'react-native-reanimated';
+import Animated, {
+  EasingNode,
+  timing,
+  interpolateColors,
+} from 'react-native-reanimated';
 import { styles } from './styles';
 
+// @ts-ignore
 import makeVisibleWhite from './assets/make_visible_white.png';
+// @ts-ignore
 import makeInvisibleWhite from './assets/make_invisible_white.png';
+// @ts-ignore
 import makeVisibleBlack from './assets/make_visible_black.png';
+// @ts-ignore
 import makeInvisibleBlack from './assets/make_invisible_black.png';
 
-export interface Props extends TextInputProps {
+export interface Props extends Omit<TextInputProps, 'secureTextEntry'> {
   /** Style to the container of whole component */
   containerStyles?: ViewStyle;
   /** Changes the color for hide/show password image */
@@ -170,7 +178,6 @@ const FloatingLabelInput: React.ForwardRefRenderFunction<InputRef, Props> = (
     currencyDivider,
     maskType,
     onChangeText,
-    secureTextEntry,
     customHidePasswordComponent,
     customShowPasswordComponent,
     isFocused,
@@ -255,23 +262,23 @@ const FloatingLabelInput: React.ForwardRefRenderFunction<InputRef, Props> = (
   );
 
   useEffect(() => {
-      if (isFocused === undefined) {
-        if (value !== '' || isFocusedState) {
-          setIsFocused(true);
-        } else if (value === '' || value === null) {
-          setIsFocused(false);
-        }
+    if (isFocused === undefined) {
+      if (value !== '' || isFocusedState) {
+        setIsFocused(true);
+      } else if (value === '' || value === null) {
+        setIsFocused(false);
       }
+    }
   }, [value]);
 
   useEffect(() => {
-      if (isFocused !== undefined) {
-        if (value !== '' || isFocused) {
-          setIsFocused(true);
-        } else {
-          setIsFocused(false);
-        }
+    if (isFocused !== undefined) {
+      if (value !== '' || isFocused) {
+        setIsFocused(true);
+      } else {
+        setIsFocused(false);
       }
+    }
   }, [isFocused, value]);
 
   useEffect(() => {
@@ -287,7 +294,7 @@ const FloatingLabelInput: React.ForwardRefRenderFunction<InputRef, Props> = (
       }
     } else {
       animateBlur();
-  }
+    }
   }, [isFocusedState]);
 
   useImperativeHandle(ref, () => ({
@@ -337,19 +344,19 @@ const FloatingLabelInput: React.ForwardRefRenderFunction<InputRef, Props> = (
           duration: animationDuration ? animationDuration : 300,
           easing: EasingNode.linear,
         }),
+        // @ts-ignore
         timing(fontColorAnimated, {
           toValue: 1,
           duration: animationDuration ? animationDuration : 300,
           easing: EasingNode.linear,
         }),
       ]).start();
-    }else if(staticLabel &&
-      isFocusedState){
-        Animated.timing(fontColorAnimated, {
-          toValue: 1,
-          duration: animationDuration ? animationDuration : 300,
-          easing: EasingNode.linear,
-        }).start();
+    } else if (staticLabel && isFocusedState) {
+      Animated.timing(fontColorAnimated, {
+        toValue: 1,
+        duration: animationDuration ? animationDuration : 300,
+        easing: EasingNode.linear,
+      }).start();
     }
   }, [halfTop]);
 
@@ -387,13 +394,14 @@ const FloatingLabelInput: React.ForwardRefRenderFunction<InputRef, Props> = (
           duration: animationDuration ? animationDuration : 300,
           easing: EasingNode.linear,
         }),
+        // @ts-ignore
         timing(fontColorAnimated, {
           toValue: 1,
           duration: animationDuration ? animationDuration : 300,
           easing: EasingNode.linear,
         }),
       ]).start();
-    }else{
+    } else {
       Animated.timing(fontColorAnimated, {
         toValue: 1,
         duration: animationDuration ? animationDuration : 300,
@@ -429,13 +437,14 @@ const FloatingLabelInput: React.ForwardRefRenderFunction<InputRef, Props> = (
           duration: animationDuration ? animationDuration : 300,
           easing: EasingNode.linear,
         }),
+        // @ts-ignore
         timing(fontColorAnimated, {
           toValue: 0,
           duration: animationDuration ? animationDuration : 300,
           easing: EasingNode.linear,
         }),
       ]).start();
-    }else{
+    } else {
       Animated.timing(fontColorAnimated, {
         toValue: 0,
         duration: animationDuration ? animationDuration : 300,
@@ -494,7 +503,6 @@ const FloatingLabelInput: React.ForwardRefRenderFunction<InputRef, Props> = (
     ? customShowPasswordImage || makeVisibleWhite
     : customHidePasswordImage || makeInvisibleWhite;
 
-
   const style: TextStyle = {
     ...setGlobalStyles?.labelStyles,
     ...labelStyles,
@@ -506,9 +514,15 @@ const FloatingLabelInput: React.ForwardRefRenderFunction<InputRef, Props> = (
       : !isFocusedState
       ? customLabelStyles.fontSizeBlurred
       : customLabelStyles.fontSizeFocused,
+    // @ts-ignore
     color: interpolateColors(fontColorAnimated, {
       inputRange: [0, 1],
-      outputColorRange: [customLabelStyles.colorBlurred,customLabelStyles.colorFocused]
+      outputColorRange: [
+        // @ts-ignore
+        customLabelStyles.colorBlurred,
+        // @ts-ignore
+        customLabelStyles.colorFocused,
+      ],
     }),
     alignSelf: 'center',
     position: 'absolute',
